taxonomy/templates/taxonomy/refresh_course_skills.html

```diff
@@ -0,0 +1,38 @@
+{% extends "admin/base_site.html" %}
+{% load i18n static %}
+
+{% block title %}Taxonomy{% endblock %}
+
+{% block branding %}
+    <h1 id="site-name">'Taxonomy Administration'</h1>
+{% endblock %}
+
+{% block nav-global %}
+    {% if user.is_staff %}
```

Comment:
Why apply the CSS for only staff users? Shouldn't this entire page be hidden behind this check?

Reply:
Also, since you are only rendering this page for `staff` users, is this check even required?